### PR TITLE
add package hip-tensor

### PR DESCRIPTION
**Pull request by afzpatel**

### CODE REVIEW

File: var/spack/repos/builtin/packages/hip-tensor/package.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    for ver in ["5.7.0", "5.7.1", "master"]:
+        depends_on("composable-kernel@" + ver, when="@" + ver)
+        depends_on("rocm-cmake@" + ver, when="@" + ver)
+
+    def setup_build_environment(self, env):
+        env.set("CXX", self.spec["hip"].hipcc)
+
+    def cmake_args(self):
+        args = []
+        return args
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/hip-tensor/package.py]: I think you can remove this:
```suggestion
```

File: var/spack/repos/builtin/packages/hip-tensor/package.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    maintainers("srekolam", "afzpatel")
+
+    version("master", branch="master")
+    version("5.7.1", sha256="96743d4e695fe865aef4097ae31d9b4e42a2d5a92135a005b0d187d9c0b17645")
+    version("5.7.0", sha256="4b17f6d43b17fe2dc1d0c61e9663d4752006f7898cc94231206444a1663eb252")
+
+    for ver in ["5.7.0", "5.7.1", "master"]:
+        depends_on("composable-kernel@" + ver, when="@" + ver)
+        depends_on("rocm-cmake@" + ver, when="@" + ver)
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/hip-tensor/package.py]: Can you use f-strings here?

**Review by alalazo — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/hip-tensor/package.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class HipTensor(CMakePackage, ROCmPackage):
+    """AMD’s C++ library for accelerating tensor primitives"""
+
+    homepage = "https://github.com/ROCmSoftwarePlatform/hipTensor"
```
Comment (srekolam) [inline var/spack/repos/builtin/packages/hip-tensor/package.py:12]: can you update the url to point to https://github.com/ROCm/hipTensor

File: var/spack/repos/builtin/packages/hip-tensor/package.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    git = "https://github.com/ROCmSoftwarePlatform/hipTensor.git"
+    url = "https://github.com/ROCmSoftwarePlatform/hipTensor/archive/refs/tags/rocm-5.7.1.tar.gz"
+    tags = ["rocm"]
```
Comment (srekolam) [inline var/spack/repos/builtin/packages/hip-tensor/package.py:15]: can you update the homepage and git and url to https://github.com/ROCm/hipTensor ; https://github.com/ROCm/hipTensor.git ; https://github.com/ROCm/hipTensor/archive/refs/tags/rocm-5.7.0.tar.gz

**Review by alalazo — APPROVED** [review]

**Pull request merged**